strings/base_error.h

```diff
@@ -179,7 +179,7 @@ namespace winrt::impl
 
     [[noreturn]] inline void __stdcall fallback_RoFailFastWithErrorContext(int32_t) noexcept
     {
-        std::terminate();
+        abort();
```

Comment:
I'm somewhat surprised that we don't have to say `std::abort()` here, but obviously it's working somehow.